src/actions/collectiblesActions.js

```diff
@@ -91,6 +92,22 @@ export const parseCollectibleFromOpenSeaAsset = (asset: OpenSeaAsset): Collectib
   };
 };
 
+export const parsePolygonCollectibleFromEtherspot = (asset: NftList): Collectible => {
+  return {
+    id: asset.items[0].tokenId,
+    name: asset.items[0].name,
+    description: asset.description,
+    icon: asset.items[0].image,
+    iconUrl: asset.items[0].image,
+    image: asset.items[0].image,
+    imageUrl: asset.items[0].image,
```

Comment:
If items is null then this will throw an error, could we add a check here that there is an array item that exists?

Reply:
yeah sure